pages/linux/pacman-remove.md

```diff
@@ -4,30 +4,30 @@
 > See also: `pacman`.
 > More information: <https://manned.org/pacman.8>.
 
-- Remove a package and its dependencies:
+- [R]emove a package and its dependencies recursively ([s]):
 
-`sudo pacman --remove --recursive {{package}}`
+`sudo pacman -Rs {{package}}`
 
-- Remove a package and both its dependencies and configuration files:
+- Remove a package and both its dependencies and configuration files ([n]):
```

Comment:
Ideally the mnemonic should be tacked after a word that's at least similar to the longform option